wiki/Beatmap_Discussion/nl.md

```diff
@@ -0,0 +1,160 @@
+---
```

Comment:
image captions not translated

Reply:
there are still some untranslated ones:
L106
L120
L130

Reply:
Sorry, ctrl+f sucks. Will add in a moment.